DfE.FindInformationAcademiesTrusts/Setup/PostBuildSetup.cs

```diff
@@ -43,5 +43,12 @@ public static void ConfigureApp(WebApplication app)
         app.MapRazorPages();
         app.UseMiddleware<ResponseHeadersMiddleware>();
         app.MapHealthChecks("/health").AllowAnonymous();
+
+        // Only setup Azure App Configuration if the Connection string is present 
+        var appConfigString = app.Configuration["AppConfig"];
+        if (!string.IsNullOrWhiteSpace(appConfigString))
```

Comment:
I think we should base this `if` on environment rather than whether or not the config is present so it doesn't silently skip if the config variable is missing when it shouldn't be. Perhaps it could be moved up to the condition on line 18?

Reply:
See https://github.com/DFE-Digital/find-information-about-academies-and-trusts/pull/485/files#r1890021967